app/code/Magento/Cms/Test/Mftf/Section/AdminBlockGridSection.xml

```diff
@@ -14,5 +14,6 @@
         <element name="checkbox" type="checkbox" selector="//label[@class='data-grid-checkbox-cell-inner']//input[@class='admin__control-checkbox']"/>
         <element name="select" type="select" selector="//tr[@class='data-row']//button[@class='action-select']"/>
         <element name="editInSelect" type="text" selector="//a[contains(text(), 'Edit')]"/>
+        <element name="gridDataRow" type="input" selector=".data-row .data-grid-cell-content"/>
```

Comment:
Please rely on data attributes instead of CSS classes/ID attribute in selectors (see http://devdocs.magento.com/guides/v2.0/coding-standards/code-standard-demarcation.html#business-logic-and-javascript).